docs/misc.rst

```diff
@@ -0,0 +1,59 @@
+.. _topics-misc:
+
+====
+Misc
+====
+
+Developer Install
+=================
+If you are are a ducktape developer, consider using the develop command instead of install. This allows you to make code changes without constantly reinstalling ducktape (see http://stackoverflow.com/questions/19048732/python-setup-py-develop-vs-install for more information)::
+
+    cd ducktape
+    python setup.py develop
+
+To uninstall::
+
+    cd ducktape
+    python setup.py develop --uninstall
+
+
+Unit Tests
+==========
+
+You can run the tests via the setup.py script::
+
+    python setup.py test
+
+Alternatively, if you've installed pytest ``sudo pip install pytest`` you can run
+it directly on the ``tests`` directory::
+
+    py.test tests
+
+
+Windows
+=======
+
+Ducktape supports Service`s that run on Windows, but only in EC2.
```

Comment:
nit: there's a backtick in `Service's` instead of an apostrophe that's mucking up with formatting